### PR TITLE
Add govuk frontend loading with scss and npm

### PR DESCRIPTION
**Pull request by 252afh**
## Context

Adding govuk frontend loading in the style Duncan implemented

## Guidance to review

Check the app loads and the govuk styling is in place

## Things to check

- [x] I have added any new ENV vars in all deployed environments
- [x] I have tested any code added or changed

### CODE REVIEW

File: django_app/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "description": "Frontend dependencies for i.AI Redbox application",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (lmwilkigov) [inline django_app/package.json:13]: Should this be MIT?

**Review by lmwilkigov — CHANGES_REQUESTED** [review]

**Comment by KevinEtchells** [comment]
Having trouble with compressor package error, so not yet able to fully review

**Review by KevinEtchells — APPROVED** [review]

**Comment by lmwilkigov** [comment]
@duncanjbrown, any thoughts on the compression error in the build here?

**Comment by duncanjbrown** [comment]
This is down to a nuance of `docker-compose`: when you mount a local volume in the `docker-compose.yml` its contents will shadow whatever was built in the `build` step.

The fix is to make sure this isn't touched is to mount a persistent volume on the node_modules folder

```
    volumes:
      - ./:/app/:z
      - /app/django_app/frontend/node_modules
```

then correct the path in style.scss, which needs to be relative to `/app` as the container runs in the context `.`:

```
@import "django_app/frontend/node_modules/govuk-frontend/dist/govuk/all";
```

and you need to correct the `Dockerfile` to this so that the node_modules folder is copied and not its contents:

```
COPY --from=node_modules /app/node_modules /app/django_app/frontend/node_modules
```

You can confirm it's built correctly with

```
docker-compose run -it django-app bash -c "ls django_app/frontend/node_modules" 
```

and you should see `govuk-frontend`

Further reading: https://stackoverflow.com/questions/29181032/add-a-volume-to-docker-but-exclude-a-sub-folder

**Pull request merged**